data/is_ocaml_yet/web.md

```diff
@@ -0,0 +1,258 @@
+---
+id: web
+question: Is OCaml web yet?
+answer: Not quite yet, but we're getting there!
+categories:
+  - name: Authentication
+    status: 🆘
+    description: |
+      n/a
+    packages:
+    - FPauth
+    - u2f
+    - webauthn
+    - jwt
+    - cookie
+    - ssl
+    - twostep
+    - oidc
+  - name: Binary protocols
+    status: 🟡
+    description: |
+      There are implementations for a lot of popular protocols, some of them are solid, but some of them are still incomplete and need more testing. Some useful protocols for which there is no package include FlatBuffers and AMQP.
+    packages:
+    - asn1-combinators
+    - avro
+    - capnp
+    - grpc
+    - protobuf
+    - rpc
+    - jsonrpc
+    - thrift
+    - bson
+    - cbor
+    - mqtt
+  - name: Browser
+    status: 🟢
+    description: |
+      OCaml has excellent support for running in the browser!
+      The [Js_of_ocaml](/p/js_of_ocaml/latest) compiler, which translates OCaml into JavaScript, is a well-established tool dating back to 2010. It integrates seamlessly with the existing OCaml ecosystem and powers numerous production applications.
+
+      In addition, [Melange](/p/melange/latest) is a more recent entry to the OCaml-JavaScript compiler space. Originating as a fork of [ReScript](https://rescript-lang.org/), Melange may not have the extensive track record of Js_of_ocaml, but it builds on a very strong foundation and already integrates well with Dune, the OCaml build system.
+    packages:
+    - js_of_ocaml
+    - melange
+    - brr
+    - gen_js_api
+  - name: Compression
+    status: 🟡
+    description: |
+      n/a
+    packages:
+    - zlib
+    - decompress
+    - camlzip
+    - lz4
+    - ezgzip
+    - hpack
+    - brotli
+    - zstd
+    - snappy
+    - lzo
+    - zstandard
```

Comment:
We probably can add [`carton`](https://github.com/mirage/ocaml-git) which does two levels of compressions (`zlib` and binary diff between objects).

Reply:
Done